nemo/collections/nlp/modules/common/megatron/transformer.py

```diff
@@ -113,10 +114,19 @@ def __init__(
         onnx_safe=False,
         activation='gelu',
         bias=True,
+        arch='transformer',
```

Comment:
We should add some hierarchy to these methods' arguments otherwise it will soon become unmanageable.  E.g. group feature related arguments into a dictionary objects.